eth/tracers/api_test.go

```diff
@@ -52,18 +56,82 @@ var (
 	errTransactionNotFound = errors.New("transaction not found")
 )
 
+type mockHistoricalBackend struct{}
```

Comment:
Adding the test utils here is ok, but we may want to revisit this to pull it into the monorepo testing instead, to keep the geth diff lean.